cg/cli/upload.py

```diff
@@ -452,3 +460,17 @@ def validate(context, family_id):
                 click.echo(f"{sample_id}: {mean_coverage:.2f}X - {completeness:.2f}%")
             else:
                 click.echo(f"{sample_id}: sample not found in chanjo", color='yellow')
+
+
+@upload.command()
+@click.option('-d', '--days', 
+            help='load X days old sampels from genotype to vogue')
+@click.pass_context
+def trending_genotype(context, days: int):
```

Comment:
So, I presume most of the vogue related transfers will be after an bioit analysis will be done, right? This function would upload as soon as as the analysis is finished and needs a `family_id` as parameter, not a "so many days ago" parameter. (Like the other functions ;))
We can then call it together with all the other upload functions from the `cg upload auto`.

We also have to come up with a better name than this. It has too many downsides: it uses an underscore, it uses two nouns. Most of the other upload functions answer the question: where are you going to upload (scout, genotype, ...) or what are you go upload (observations)? e.g. genotype-trends would work a lot better. If vogue is going to have more actions, why not add `cg upload trends` or `cg upload trends genotype` and even a 'cg upload trends all` (if that is even possible).

Reply:
Like it is built, it can run every day and fetch what ever was updated in the database witin the last day eg. Or every second day, or whatever. Is that a problem? Otherwise Ill have to redo the genotype PR as well. Is it worth the amount of time? Would have been nice to get this kind of restrictions before starting to develop for the cg cli.

"cg uploads trends genotype"  I like! 👍

Reply:
Like discussed, let's drop the rewriting unless you really feel like it ;)